virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,73 @@
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Red Hat Enterprise Linux virtual machines] using the wizard.
+* Create a Microsoft Windows virtual machine
+** xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Create and upload a Microsoft Windows boot source].
+** Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Microsoft Windows virtual machines] using the wizard.
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a virtual machine using xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-to-the-terminal_virt-accessing-vm-consoles[the terminal] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[the serial console] in the {product-title} web console.
+* Connect to a virtual machine using SSH:
+** xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[Create a service] to expose a route for the virtual machine.
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+
+Manage a virtual machine.
+
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Network options:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[Configure a Linux bridge network], in addition to the default pod network.
+* Install the xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[SR-IOV Operator], configure a xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[SR-IOV network device], and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[attach a virtual machine to the SR-IOV network].
+
+Microsoft Windows virtual machine options:
+
+* xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows virtual machine installations with `sysprep`].
+* Connect to a Microsoft Windows virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-rdp-console_virt-accessing-vm-consoles[RDP].
+
+Monitoring {VirtProductName}
```

Comment:
Suggest changing this to "Monitor your {virt} environment:", just for consistency.

Reply:
OK, now I'm confused.... you had said above that you didn't want a section with the one task ---- let me know if that needs to be changed back.

Reply:
Done.